MaskRCNN/model/generalized_rcnn.py

```diff
@@ -116,6 +117,18 @@ def backbone(self, image):
 
 
     def rpn(self, image, features, inputs, orig_image_dims):
+        """
+        The RPN part of the graph that generate the RPN proposal and losses
+
+        Args:
+            image: BS x Num_channel x H x W
+            features: ([tf.Tensor]): A list of 5 FPN feature level P23456, each with BS x NumChannel x H x W
```

Comment:
We are using 'Num_channel`/`NumChannel`, `H` and `W` to refer to multiple values. Image is `BS x Num_Channel x H x W`, while features are `BS x NumChannel x fH x fW`. We need to clarify that these variables are not the same and choose a consistent format for `Num_channel` vs `NumChannel`